src/app/berkeley_migration/berkeley_migration.ml

```diff
@@ -567,6 +588,13 @@ let () =
            Param.flag "--network" ~aliases:[ "-network" ]
              Param.(required string)
              ~doc:"Network name used when downloading precomputed blocks"
+         and stream_precomputed_blocks =
```

Comment:
Please also expose this (and probably keep-precomputed-blocks) in scripts/archive/migration/berkeley_migration.sh